.github/workflows/ci-linux.yml

```diff
@@ -0,0 +1,70 @@
+name: Build/test on Linux
+# This workflow is triggered on pushes to the repository.
+on: [push, pull_request]
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                py: [
+                    '2.7',
+                    '3.5',
+                    '3.6',
+                    '3.7',
+                    '3.8',
+                    'pypy2',
+                    'pypy3'
+                ]
+        name: "Python: ${{ matrix.py }}"
+        steps:
+            - uses: actions/checkout@master
+            - name: Setup python
+              uses: actions/setup-python@v1
+              with:
+                  python-version: ${{ matrix.py }}
+                  architecture: x64
+            - run: pip install tox
+            - name: Running tox
+              run: tox -e py
+    coverage:
+        runs-on: ubuntu-latest
+        name: Validate coverage for Python 2/3
+        steps:
+            - uses: actions/checkout@master
+            - name: Setup python
+              uses: actions/setup-python@v1
+              with:
+                  python-version: 2.7
+                  architecture: x64
+            - name: Setup python
+              uses: actions/setup-python@v1
+              with:
+                  python-version: 3.8
+                  architecture: x64
+            - run: pip install tox
+            - run: tox -e py38-pyramid110,py27,coverage
```

Comment:
Note that this does not match the other CI files I've added to various other Pylons projects... cause we need `pyramid_retry` installed to validate full coverage.